src/app/editor-tabs/editor-tabs.component.spec.ts

```diff
@@ -113,4 +125,62 @@ describe('EditorTabsComponent', () => {
     expect(activeItem.nativeElement.innerText).toBe("foo");
   });
 
+  it('emits editor.active event on navigation.open event', () => {
+    // when
+    messagingService.publish(events.NAVIGATION_OPEN, fooDocument);
+
+    // then
+    expect(editorActiveCallback).toHaveBeenCalledTimes(1);
+    expect(editorActiveCallback).toHaveBeenCalledWith({ path: fooDocument.path });
+  });
+
+  it('emits editor.active event on tab switch', () => {
+    // given
+    openFooAndBar();
+    editorActiveCallback.calls.reset();
+    let navItems = tabset.queryAll(By.css('.nav-item > a'));
```

Comment:
That's a CSS selector and means "Select every <a> element where the parent is of class .nav-item"

https://www.w3schools.com/cssref/sel_element_gt.asp